deepwell/src/endpoints/locale.rs

```diff
@@ -58,18 +58,29 @@ pub async fn translate_strings(
     ctx: &ServiceContext<'_>,
     params: Params<'static>,
 ) -> Result<TranslateOutput> {
-    let TranslateInput {
-        locale: locale_str,
-        messages,
-    } = params.parse()?;
+    let TranslateInput { locales, messages } = params.parse()?;
+
+    if locales.is_empty() {
+        error!("No locales specified in translate call");
+        return Err(ServiceError::NoLocalesSpecified);
+    }
 
     info!(
-        "Translating {} message keys in locale {locale_str}",
+        "Translating {} message keys in locale {} (or {} fallbacks)",
         messages.len(),
+        &locales[0],
+        locales.len() - 1,
     );
 
-    let locale = LanguageIdentifier::from_bytes(locale_str.as_bytes())?;
     let mut output: TranslateOutput = HashMap::new();
+    let locales = {
+        let mut langids = Vec::new();
+        for locale in locales {
+            let langid = LanguageIdentifier::from_bytes(locale.as_bytes())?;
+            langids.push(langid);
+        }
+        langids
+    };
```

Comment:
Forgot this was used here, I should move the helper function below out somewhere public so it can get imported and used here too.